clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/handlers/BasicAmazonDeployHandler.groovy

```diff
@@ -251,6 +253,12 @@ class BasicAmazonDeployHandler implements DeployHandler<BasicAmazonDeployDescrip
         description.spotPrice = null
       }
 
+      def capacity = new DeploymentResult.Deployment.Capacity(
+          min: description.capacity.min != null ? description.capacity.min : 0,
```

Comment:
isn't that what the elvis operator is for?

`min: description.capacity.min ?: 0,`